block-storage/src/test/scala/coop/rchain/blockstorage/BlockGen.scala

```diff
@@ -18,26 +18,22 @@ object BlockGen {
   def distinctListOf[T: Arbitrary] =
     distinctListOfGen(arbitrary[T])(_ == _)
 
-  val blockHashGen: Gen[BlockHash] = for {
-    byteArray <- listOfN(32, arbitrary[Byte])
-  } yield ByteString.copyFrom(byteArray.toArray)
-
-  implicit val arbitraryHash: Arbitrary[BlockHash] = Arbitrary(blockHashGen)
+  def byteStringGen(size: Int): Gen[ByteString] =
+    for {
+      byteArray <- listOfN(size, arbitrary[Byte])
+    } yield ByteString.copyFrom(byteArray.toArray)
 
   val blockElementGen: Gen[BlockMessage] =
     for {
-      hash      <- arbitrary[BlockHash]
-      validator <- arbitrary[Validator]
+      hash      <- byteStringGen(32)
+      validator <- byteStringGen(32)
```

Comment:
Might duplicate https://github.com/rchain/rchain/pull/2129 but I guess it is fine for now.